tf_agents/utils/nest_utils.py

```diff
@@ -556,3 +556,26 @@ def where(condition, true_outputs, false_outputs):
   """
   return tf.nest.map_structure(lambda t, f: tf.compat.v1.where(condition, t, f),
                                true_outputs, false_outputs)
+
+def wherev2(condition, true_outputs, false_outputs):
+  """Generalization of tf.where where supporting nests as the outputs.
```

Comment:
and inputs?

Reply:
Yes, in that the inputs are the nests from which tf.where selects elements to construct the output. I figured this is sort of implicit in the nature of "where" (and, honestly, I copied the docstring from the existing nest_utils.where() :) ). 

I will update the docstring to be clearer.

Reply:
Clarified.